src/js/components/Breadcrumbs.js

```diff
@@ -1,186 +1,18 @@
-import classNames from 'classnames';
-import DeepEqual from 'deep-equal';
 import mixin from 'reactjs-mixin';
 /* eslint-disable no-unused-vars */
 import React, {PropTypes} from 'react';
 /* eslint-enable no-unused-vars */
-import ReactDOM from 'react-dom';
 import {StoreMixin} from 'mesosphere-shared-reactjs';
 
-import BreadcrumbSegmentLink from './BreadcrumbSegmentLink';
-import IconChevron from './icons/IconChevron';
-
-const COLLAPSE_BUFFER = 12;
-const LAST_ITEM_OFFSET = 150; // Difference between scrollWidth and outerWidth
-const PADDED_ICON_WIDTH = 38; // Width of icon + padding
+import ManualBreadcrumbs from './ManualBreadcrumbs';
 
 class Breadcrumbs extends mixin(StoreMixin) {
```

Comment:
i'd like to rename this to `RouteBreadcrumbs` but then that can probably go inside of a separate PR